WaterConsumption/examples/example.jsonld

```diff
@@ -0,0 +1,138 @@
+{
+    "id": "urn:ngsi-ld:Consumer:Consumer139E643",
+    "type": "Consumer",
+    "https://ontology.eglobalmark.com/WaterSmartMeter#acquisitionStageFailure": {
+        "type": "Property",
+        "observedBy": {
```

Comment:
Although accepted the observedAt and observedBy does not provide much value to be repeated in all properties.

Reply:
From what I was told, we (Exeter) did not create the waterconsumption object and therefore is hesistant to change its set-up.